packages/client/src/components/StoryPointingVotes.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+export const StoryPointingVotes = ({ users, meetingUsers, votes, showVotes }) => {
+  return (
+    <>
+      <h3>Users Votes:</h3>
+      {meetingUsers.length ? (
+        <>
+          {meetingUsers.map(userId => {
+            const meetingUser = users[userId];
+            const userVote = votes[userId];
+
+            return (
+              <p key={userId}>
+                {meetingUser.name}: {!!userVote ? (showVotes ? userVote : '###') : ''}
+              </p>
+            )
+          })}
+        </>
+      ) : (
+        <p>
+          <i>No users yet</i>
+        </p>
+      )}
+    </>
+  );
+}
```

Comment:
Prefer early returns
```suggestion
export const StoryPointingVotes = ({ users, meetingUsers, votes, showVotes }) => {
  if (!meetingUsers.length) {
    return (
      <p>
        <i>No users yet</i>
      </p>
      )
  }
  
  return (
    <>
      <h3>Users Votes:</h3>
          {meetingUsers.map(userId => {
            const meetingUser = users[userId];
            const userVote = votes[userId];
            return (
              <p key={userId}>
                {meetingUser.name}: {!!userVote ? (showVotes ? userVote : '###') : ''}
              </p>
            )
          })}
    </>
  );
}
```

Reply:
In this case it always show the section title, plus the list of users or the empty state.
I'll need to add the title on the early return as well. To avoid code duplication I'll create a method to get the title.